marti_common_msgs/msg/HealthStatus.msg

```diff
@@ -0,0 +1,12 @@
+# This message is intended for use as a light-weight machine status indicator for robot consumption
+# whereas Diagnostics are for human consumption
+
+int8 OK=0    # Initialized with no issues
```

Comment:
My only reservation about that is it is unclear what type byte actually is. Is it signed or unsigned?